cli/waiter/subcommands/kill.py

```diff
@@ -0,0 +1,99 @@
+import logging
+
+from tabulate import tabulate
+
+from waiter import http_util, terminal
+from waiter.format import format_timestamp_string
+from waiter.querying import query_services, services_using_token
+from waiter.util import guard_no_cluster, str2bool, response_message, print_error, wait_until
+
+
+def kill_service_on_cluster(cluster, service_id, token_name):
+    """Kills the service with the given service id in the given cluster."""
+
+    def service_is_killed():
+        return service_id not in [s['service-id'] for s in services_using_token(cluster, token_name)]
+
+    cluster_name = cluster['name']
+    try:
+        print(f'Killing service {terminal.bold(service_id)} in {terminal.bold(cluster_name)}...')
+        resp = http_util.delete(cluster, f'/apps/{service_id}')
+        logging.debug(f'Response status code: {resp.status_code}')
+        if resp.status_code == 200:
+            killed = wait_until(service_is_killed, timeout=30, interval=1)
+            if killed:
+                print(f'Successfully killed {service_id} in {cluster_name}.')
+                return True
+            else:
+                print_error('Timeout waiting for service to die.')
+                return False
+        else:
+            print_error(response_message(resp.json()))
+            return False
+    except Exception:
+        message = f'Encountered error while killing {service_id} in {cluster_name}.'
+        logging.exception(message)
+        print_error(message)
+
+
+def format_status(status):
+    """Formats service status"""
+    if status == 'Running':
```

Comment:
Nitpick: This running is a config on Waiter from `:messages :service-state-running`. Do we care about these changing? If so, we should load the string from there in the `/settings` endpoint.

Reply:
I would say it's OK to rely on these not changing